Os/CMakeLists.txt

```diff
@@ -96,7 +95,11 @@ if (FPRIME_USE_BAREMETAL_SCHEDULER)
     list(APPEND SOURCE_FILES "${CMAKE_CURRENT_LIST_DIR}/Baremetal/SystemResources.cpp")
 endif()
 register_fprime_module()
+
 require_fprime_implementation(Os/File)
+# require_fprime_implementation(Os/Task) # should be added in
```

Comment:
Uncommenting this along with the `Os` MOD_DEPS was causing errors - I haven't tracked them down so I kept it commented out here for further exploration

Reply:
@LeStarch I believe I was able to fix by removing the Os/Task includes and adding some Fw/Time link dependencies, see https://github.com/nasa/fprime/pull/2790/commits/7a27ec9238631e39a5b71688bc531322631c9e3a
Please confirm this is ok

Reply:
Ugh - lots of new errors popping in CI haha

Reply:
Reverting is fine, we can tracks as a cleanup item